Sources/FluentUI_macOS/Components/MultilinePillPicker/MultilinePillPickerView.swift

```diff
@@ -49,6 +49,6 @@ public final class MultilinePillPickerView: ControlHostingView {
 	private func updatePicker() {
 		let picker = MultilinePillPicker(labels: labels, action: action)
```

Comment:
Does this approach block SwiftUI from nicely handling the state updates with transitions? Specifically `isEnabled` and `labels` feel like they should be Published props in an `ObservableObject`.

Reply:
It's a good point! That's actually how most of our Fluent iOS components work. I'm not sure how I'd want to arrange that though -- the approach we used there is a bit cumbersome and doesn't play well with established things like pulling `isEnabled` from the environment.

Reply:
Talked offline, and came up with a very clever solution. Thanks @lukas-cap!